packages/gatsby-plugin-hooked-head/gatsby-ssr.js

```diff
@@ -0,0 +1,18 @@
+import { createElement } from 'react';
+import { toStatic } from 'hooked-head';
+
+export const onRenderBody = ({ setHeadComponents, setHtmlAttributes }) => {
+  const { title, metas, lang } = toStatic();
+
+  if (lang && setHtmlAttributes) {
```

Comment:
```suggestion
  if (lang && setHtmlAttributes) {
```

could be

```
if (lang)
```

I don't think `setHtmlAttributes` can ever be a falsy.

Reply:
Thanks! Really awesome dogfooding, converted my own portfolio yesterday as well! Your enthusiasm has been really motivating 🙏

Reply:
@JoviDeCroock Nice! 4kb less here (hooked-head), 10kb less there (emotion -> linaria). It adds up :)

Thank YOU. Had quite a lot of fun with this the last two days. :1st_place_medal: